extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/GuacamoleTunnelService.java

```diff
@@ -145,4 +148,39 @@ GuacamoleTunnel getGuacamoleTunnel(AuthenticatedUser user,
      */
     public Collection<ActiveConnectionRecord> getActiveConnections(ConnectionGroup connectionGroup);
 
+    /**
+     * Creates a socket for the given user which joins the given active
+     * connection. The given client information will be passed to guacd when
+     * the connection is established. This function will apply any concurrent
+     * usage rules in effect, but will NOT test object- or system-level
```

Comment:
"object-" is a bit funny looking. Maybe just "object or system-level"? Or "object-level or system-level"?